src/core/Akka/Util/Internal/ArrayExtensions.cs

```diff
@@ -27,6 +27,16 @@ public static bool IsNullOrEmpty(this Array obj)
         }
 
         /// <summary>
+        /// Determines if an array is not null or empty.
+        /// </summary>
+        /// <param name="obj">The array to check.</param>
+        /// <returns>True if not null or empty, false otherwise.</returns>
+        public static bool NonEmpty(this Array obj)
```

Comment:
Wouldn't linq `.Any()` work for this? I just realized that right after I closed the PR :\